src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,67 @@
+import {Component} from 'react';
+import React from 'react';
+import * as AuthActions from "../../store/Auth/actions";
+import Alert from "react-s-alert";
+import {ACCESS_TOKEN} from "../../services/constants";
+import * as LoginFormActions from '../../store/LoginForm/actions';
+import {connect} from 'react-redux';
+import * as LoginSelectors from '../../store/LoginForm/reducer'
+
+/**
+ * Login form for input password, email, e.t.c
+ */
+class LoginForm extends Component {
+    constructor(props) {
+        super(props);
+        this.handleInputEmail = this.handleInputEmail.bind(this);
```

Comment:
You can delete this constructor and add the binding of all methods instead of explicitly calling them, for example you see below this line in render method

Reply:
in general, it is better to combine all these entries into one method and through the attribute in tags for which the input change event is set (for later insertion into a container with data) to get a value that can be directly inserted into the data container
in render function:
```javascript
render = () => {
    return(
      ///....
      <input type="email" name="email"
                           className="form-control" placeholder="Email"
                           value={this.props.email} onChange={this.handleChange.bind(this)} required/>
       ///....
   )
}
```

```javascript

handleChange = (event) => {
    event.preventDefault();
    let action = event.target.getAttributet("action");
   if (action !== null){
     let data = event.target.value;
     handleSubmitData(action, data);
     return;
   }
}

handleSubmitData(action, data){
   switch(action){
      case "email":
        this.props.LoginFromActions.inputEmail(data);
        return
     ///... and e.t.c
    }
}
```